link/src/main/java/com/stripe/android/link/ui/wallet/WalletModals.kt

```diff
@@ -15,13 +16,42 @@ import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import com.stripe.android.link.R
+import com.stripe.android.link.model.removeConfirmation
+import com.stripe.android.link.model.removeLabel
 import com.stripe.android.link.theme.HorizontalPadding
 import com.stripe.android.link.theme.linkColors
+import com.stripe.android.model.ConsumerPaymentDetails
+
+private val BottomSheetFirstItemModifier = Modifier.padding(
+    start = HorizontalPadding,
+    top = 24.dp,
+    end = HorizontalPadding,
+    bottom = 10.dp
+)
+
+private val BottomSheetMiddleItemModifier = Modifier.padding(
+    horizontal = HorizontalPadding,
+    vertical = 10.dp
+)
+
+private val BottomSheetLastItemModifier = Modifier.padding(
+    start = HorizontalPadding,
+    top = 10.dp,
+    end = HorizontalPadding,
+    bottom = 24.dp
+)
 
 @Preview
 @Composable
 internal fun WalletBottomSheetContent() {
     WalletBottomSheetContent(
+        paymentDetails = ConsumerPaymentDetails.BankAccount(
+            id = "id",
+            isDefault = true,
+            bankIconCode = null,
+            bankName = "Bank Name",
+            last4 = "last4"
```

Comment:
Should these be localized

Reply:
This is a @Preview, just for visualization